go/test/endtoend/reparent/plannedreparent/reparent_test.go

```diff
@@ -557,7 +558,16 @@ func waitForFilePosition(t *testing.T, clusterInstance *cluster.LocalProcessClus
 
 // fileNameFromPosition gets the file name from the position
 func fileNameFromPosition(pos string) string {
-	return pos[0 : len(pos)-4]
```

Comment:
Wow, this was assuming there were 3 position digits this whole time? 🙂

Reply:
@mattlord yeah strange! 🙈